### PR TITLE
kaizen: vanity import path

### PR DESCRIPTION
**Pull request by timbray**
Worth mentioning that the path is: quamina.net/go/quamina

Signed-off-by: Tim Bray <tbray@textuality.com>

### CODE REVIEW

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         id: go
 
       - name: Run benchmark
-        run: go test  -benchmem -run="^$" -bench "^Benchmark" . github.com/timbray/quamina | tee output.txt
+        run: go test  -benchmem -run="^$" -bench "^Benchmark" . quamina.net/go/quamina | tee output.txt
```
Comment (embano1) [inline .github/workflows/benchmarks.yml:32]: Question: do we need the go mod path here at all? I think this is enough:

```suggestion
        run: go test  -benchmem -run="^$" -bench "^Benchmark" . | tee output.txt
```

Or do you remember why the module path was added there?

**Review by embano1 — APPROVED** [review]
Just one question on the benchmark, otherwise LGTM

**Comment by embano1** [comment]
@timbray to fix the "out-of-date" warning please use this (non-default) option

image

Quiet nice and easy instead of doing this locally.

**Comment by embano1** [comment]
Question: Your current redirect returns:

```console
<meta name="go-import" content="quamina.net/go/quamina git https://github.com/timbray/quamina">
```

IMHO we should follow this guidance:

```
When using modules, an additional variant of the go-import meta tag is recognized and is preferred over those listing version control systems. That variant uses "mod" as the vcs in the content value, as in:
https://pkg.go.dev/cmd/go#hdr-Remote_import_paths
```

File: .github/workflows/benchmarks.yml (hunk 1 above)
Comment (timbray) [reply .github/workflows/benchmarks.yml:32]: No idea, sorry. This was done by @yosi I think?

Comment (timbray) [reply .github/workflows/benchmarks.yml:32]: I want to get this merged so I think I'll leave this in for now post a question to @yosiat and see if we can take this out at some future point

Comment (timbray) [reply .github/workflows/benchmarks.yml:32]: (I mean, the benchmarks tests are passing and previously they weren't so I want to be careful)

**Comment by timbray** [comment]
> IMHO we should follow this guidance:
> 
> ```
> When using modules, an additional variant of the go-import meta tag is recognized and is preferred over those listing version control systems. 
> That variant uses "mod" as the vcs in the content value, as in:
> 
> <meta name="go-import" content="example.org mod https://code.org/moduleproxy">
> This tag means to fetch modules with paths beginning with example.org from the module proxy available at the URL https://code.org/moduleproxy. See https://golang.org/ref/mod#goproxy-protocol for details about the proxy protocol.

But that would imply that quamina.net should implement the GOPROXY protocol, and have to respond to five different endpoints. So I would actually have to build a real website there at quamina.net so that it would track versions and provide lists and so on.  If the current simplified form actually works it's 

Or am I missing something?


> 
> Source: https://pkg.go.dev/cmd/go#hdr-Remote_import_paths
> ```

**Pull request merged**